hooks/git_commit_standards_hook.py

```
@@ -91,23 +123,51 @@ def main():
+            
+            if current_branch in ["main", "master", "production"]:
+                print(json.dumps({
+                    "decision": "block", 
+                    "reason": (
+                        "❌ Don't merge directly into protected branches!\n\n"
+                        "Use Pull Requests for merging into main/master.\n"
+                        "This ensures code review and CI/CD checks."
+                    )
+                }))
+                return 1
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)